src/platform/packages/private/kbn-index-editor/src/components/query_bar.tsx

```diff
@@ -34,13 +40,20 @@ export const QueryBar = () => {
     indexUpdateService.indexCreated$,
     indexUpdateService.isIndexCreated()
   );
+  const indexName = useObservable(indexUpdateService.indexName$, null);
 
   const [queryError, setQueryError] = useState<string>('');
 
   const discoverLocator = useMemo(() => {
     return share?.url.locators.get('DISCOVER_APP_LOCATOR');
   }, [share?.url.locators]);
 
+  const isOpeningInNewTabEnabled = useMemo(
+    () => featureFlags?.getBooleanValue('discover.tabsEnabled', true) ?? true,
```

Comment:
Are we tracking this somewhere? I understand why you didnt add the constant from here `src/platform/plugins/shared/discover/public/constants.ts` but we might miss it when the discovery team removes the flag. 

@davismcphee is there any issue already for this to add a comment?